### DISCUSSION TITLE
Is there a 'fix' option?

### DISCUSSION THREAD

**Discussion by bbannon** [linthtml/linthtml#451] [Q&A] [answered]
Is there a wat to fix things such as doctype and indent? And maybe other things.

**Comment by KamiKillertO** [comment, marked as answer]
There's no fix option at the moment, but I'm planning on adding it in a next version. I wanted to add the option it in the v0.9, but the the works required to do it was more than I've expected. So I've rescheduled the feature for an other version (v0.10 or v0.11)

**Reply by KamiKillertO** [reply]
I planned on adding it in the upcoming version but could not because the current html parser ecosystem is too "opinionated".
I will try to add it in the v0.12

**Comment by bbannon** [comment]
Have you looked at php_cs_fixer? 
I installed it from php composer with "require friendsofphp/php-cs-fixer". 
It handles formatting php pretty well.  Lots of options.
It looks to me like a subset of that could be used for html formatting; either alone or inside php.
I don't have the time or skill to do it myself.
